digitalbiblesociety/dbp-reader
# FCBH 1490: SPIKE: DBP-Web: Oral Bible Translated languages won't play multi-file-set chapters

SPIKE: DBP-Web: Oral Bible Translated languages won't play multi-file-set chapters

Fix audio Player on Chrome Firefox and Edge. ([FCBH-1490](https://fullstacklabs.atlassian.net/browse/FCBH-4190))

HSL library has been added to support m3u8 files on Chrome, edge, and firefox. 
Same behavior for Safari.


How To Test it:

1- Run the app 
2-Access to https://newdata.bible.is/bible/mjxbsf/luk/7?audio_type=audio on:  Chrome / Firefox / Safari / Edge
3-Click on the "Play" button


Expected behavior:

The audio should play properly
